src/IRrecv.cpp

```diff
@@ -810,6 +810,10 @@ bool IRrecv::decode(decode_results *results, irparams_t *save,
     DPRINTLN("Attempting Carrier 40bit decode");
     if (decodeCarrierAC40(results, offset)) return true;
 #endif  // DECODE_CARRIER_AC40
+#if DECODE_CARRIER_AC48
+    DPRINTLN("Attempting Carrier 48bit decode");
+    if (decodeCarrier_AC48(results, offset)) return true;
```

Comment:
do we want this to be named `CarrierAC48` rather than `Carrier_AC48` ?

Reply:
When in Rome, do as the Romans do. i.e. use the same style as existing for that protocol.
e.g. `decodeCarrierAC48` as `decodeCarrierAC40` is named that way.

Reply:
My bad, checked the constants which are `CarrierAc*` functions are `CarrierAC*`
Made a rename of everything from `Carrier_AC48` to  `CarrierAc48`, need to revisit the functions again.

Reply:
Don't feel bad. It's a crappy code evolution thing. Some protocols pre-date the naming "style" and well .. I don't have the guts to break other peoples code by renaming functions/classes etc.

If I was starting again, I'd a) come up with a better naming style/scheme, b) standardise on it earlier.

Some things got inherited from the original `IRremote` library. Some things evolved poorly.
e.g. HitachiAC2 etc. I later realised adding the nr. of bits to the name worked best. (except when the protocol has variable bit lengths, which you find out _later_ ... sigh)
Some are just major brainfades on my part. e.g. `CarrierAc48` vs. `Carrier48Ac` vs. `Carrier48` .. I'm not consistent protocol to protocol. (i.e. I suck ;-)

We should probably come up with a proper naming style guide at some point, and you can hit me each time I violate it. Maybe I'll learn that way! :-)